### ISSUE TITLE
Warning: Cannot declare class PHPUnit_Framework_Assert, because the name is already in use.

### ISSUE THREAD

**Issue by theutz** [Codeception/Codeception#4479]
#### What are you trying to achieve?
Run unit tests generated by Codeception

#### What do you get instead?
A very annoying warning message

> Provide console output if related. Use `-vvv` mode for more details.

```bash
Warning: Cannot declare class PHPUnit_Framework_Assert, because the name is already in use in /Users/<me>/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/shim.php on line 17

Call Stack:
0.0003     373480   1. {main}() /Users/<me>/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/codecept:0
0.0005     401560   2. require_once('/Users/<me>/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/autoload.php') /Users/michaelutz/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/codecept:7
0.0087    1765936   3. include_once('/Users/<me>/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/shim.php') /Users/michaelutz/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/autoload.php:45
0.0103    2021592   4. class_alias() /Users/<me>/Code/theutz/advanced-custom-enums/vendor/codeception/codeception/shim.php:17
```

### Details

* Codeception version: 2.3.5, PHP Unit 6.2.4
* PHP Version: 7.0.22
* Operating System: Mac OSX Sierra 10.12.6
* Installation type: Composer

**Comment by ddinchev** [comment]
Are you using the latest `Codeception\Verify`? This hit me earlier as well, it's caused by https://github.com/Codeception/Verify/pull/31 - for now I've fixed my `Verify` version to `0.3.3`.

The problem comes from https://github.com/Codeception/Codeception/blob/2.3/shim.php#L16 - Codeception only checks `if (!class_exists('PHPUnit_Framework_TestCase') && class_exists('PHPUnit\Framework\TestCase'))` however, it creates aliases for all PHPUnit 6 classes used by Codeception. However, in Verify, the some is done for different class:
https://github.com/Codeception/Verify/blob/master/src/Codeception/function.php#L2

In my opinion, either all classes should be checked before aliased, or at least the ones aliased with `Verify`. Apparently `Verify` is used without `Codeception`, funnily it breaks `Codeception` when used together.

**Comment by matthib** [comment]
I've actually run into the same problem. In my case the Codeception tests are included in a phing build process and this issue causes the build to fail.

Gladly the fix mentioned by @ddinchev - setting the Verify version to 0.3.3 - works fine for me. But I agree that all classes should be checked before creating an alias for them in the shim file.
